account_financial_risk_ext/models/res_partner.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2019 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    credit_balance = fields.Monetary(compute="_compute_risk_exception")
+
+    @api.multi
+    @api.depends(lambda x: x._get_depends_compute_risk_exception())
```

Comment:
Recently I have been wondering what is the purpose of adding `@api.depends` to the compute method of a non-stored compute field.

Reply:
@TimLai125 From my observation, it seems that adding `@api.depends` gives you the benefit of computing the field value on the spot, whereas, without it, you need to wait until you save the record to trigger the computation.  Can you confirm this behavior when you get the chance?

Reply:
Let me check it, it could be a nice hint to one of an old SST ticket